lib/dogma/util/script_strings.ex

```diff
@@ -7,98 +7,106 @@ defmodule Dogma.Util.ScriptStrings do
   for Elixir code and reporting errors when there should be none.
   """
 
+  @sigil_delimiters [{"|", "|"}, {~S("), ~S(")}, {"'", "'"}, {"(", ")"},
+                      {"[", "]"}, {"{", "}"}, {"<", ">"}]
+  @all_string_sigils @sigil_delimiters
+                      |> Enum.flat_map(fn({b, e}) ->
+                          [{"~s#{b}", e}, {"~S#{b}", e}]
```

Comment:
I wasn't sure which ones to include. Basically, there seem to be two alternatives: 

1. Include only s/S, so the module does what it says it does ("Takes a source string and return it with all of the string literals stripped of their contents.") or
2. include all the standard sigils (c,r,s,w).

So you basically have to decide whether to be correct or practical :wink: But that is for you to decide, I just implemented what the docstring said.